032-MLOpsFromScratch/Coach/00-prereqs.md

```diff
@@ -0,0 +1,36 @@
+# Challenge 0: Pre-requisites
+
+**[Home](../README.md)** - [Next Challenge >](./01-TimeSeriesForecasting.md)
```

Comment:
Change this Home link to your new Coach README, not the top level student README

Reply:
Done.